scripts/draft-release.sh

```diff
@@ -15,7 +15,10 @@ fi
 FOLDER="${2-}"
 
 echo "Drafting release"
-github-release release --user storj --repo storj --tag "$TAG" --draft
+current_major_release_version=$(echo "$TAG"| cut -d '.' -f 1-2)
+previous_release_version=$(git describe --tags `git rev-list --exclude='*rc*' --exclude=$current_major_release_version* --tags --max-count=1`)
+changelog=$(python3 changelog.py "$TAG" "$previous_release_version" 2>&1)
```

Comment:
If there is any kind of error, we would just put this into the draft.
Acceptable for the first version, but we likely want to come up with some sort of error handling here.

Reply:
I agree, we can test it first and then add something better, anyway it will make draft and we need to review it manually